### ISSUE TITLE
Error at run_spp.R

### ISSUE THREAD

**Issue by cjhong** [ENCODE-DCC/chip-seq-pipeline2#94]
**Describe the bug**
A clear and concise description of what the problem is.
I followed the instruction at https://github.com/ENCODE-DCC/chip-seq-pipeline2 and successfully installed the pipeline. I have a single end ChIPSeq and control FASTQ file. The process from Fastq to tagAlign was good but I got error at running "run_spp.R".

**OS/Platform and dependencies**
- OS or Platform: [e.g. Ubuntu 16.04, Google Cloud, Stanford Sherlock/SCG cluster, ...]
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: Fedora
Description:    Fedora release 26 (Twenty Six)
Release:        26
Codename:       TwentySix
Linux lri-107577 4.15.6-200.fc26.x86_64 #1 SMP Mon Feb 26 18:51:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

- Conda version: If you have used Conda (`$ conda --version`).
conda 4.7.10

- Caper version
0.4.1

- If not using Caper
  - Cromwell/dxWDL version: [e.g. `cromwell-42.jar`, `dxWDL-78.jar`]

**Attach error logs**
Caper users:
Caper automatically run troubleshooter for failed workflows. If not and If you ran a Caper server and then get workflow_id of your failed workflow with `caper list`. Or directory use a `metadata.json` file on Cromwell's output directory
```
$ caper troubleshoot [WORKFLOW_ID_OR_METADATA_JSON_FILE]

The output log file can be found at 

https://drive.google.com/open?id=1iDKjGAlW1duK85MoPACkIeH9opiAKznr

In summary,
/bin/bash: line 1: 125412 Segmentation fault      (core dumped) Rscript --max-ppsize=500000 $(which run_spp.R) -rf -c=16_05T9_00S9CCF_HCT116-AHT_TotalRNAPol2_hs_i95.merged.trim_50bp.filt.no_chrM.15M.tagAlign.gz -p=2 -filtchr="chrM" -savp=16_05T9_00S9CCF_HCT116-AHT_TotalRNAPol2_hs_i95.merged.trim_50bp.filt.no_chrM.15M.cc.plot.pdf -out=16_05T9_00S9CCF_HCT116-AHT_TotalRNAPol2_hs_i95.merged.trim_50bp.filt.no_chrM.15M.cc.qc -x=-500:100

**Comment by leepc12** [comment]
This looks like a version mismatch of your R and R-packages. Do you have a R installed globally or on your main Conda env? Run the following for debugging.
```
$ conda activate encode-chip-seq-pipeline

$ which conda
$ which R
$ R --version

$ echo $R_HOME
$ echo $R_LIBS
```

**Comment by leepc12** [comment]
Closing this due to long inactivity.
